README.md

```diff
@@ -593,6 +594,21 @@ All query executions happen remotely from the **kdb VS Code extension** either a
 
   ![kdb results view](https://github.com/KxSystems/kx-vscode/blob/main/img/kdbview-results.png?raw=true)
 
+## AxLibraries
+
+The following features of [AxLibraries](https://code.kx.com/developer/libraries/) are integrated and can be used within the extension:
+
+- Grammar of Graphics
+- q static linter
+
+**Note:** To use those features, you should install AxLibraries following the [installation instructions.](https://code.kx.com/developer/libraries/#installation)
+
+### Grammar of Graphics
+
+Calling `.qp.display` displays the plot locally.
```

Comment:
@ecmel Should we have more detail here on what you need to do to get ggplots working?

Reply:
@LizNorris The link to the installation instructions are added in the Note section below.